cmd/ears/ears.yaml

```diff
@@ -27,6 +27,11 @@ ears:
       #type: dynamodb
       region: us-west-2
       tableName: bw.ears.routes
+    fragment:
+      #type: inmemory
```

Comment:
I think we should commit inmemory instead of dynamodb in config

Reply:
ok